src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -249,6 +249,21 @@ public function forceFill(array $attributes)
         });
     }
 
+    /**
+     * Qualify the given column name by the model's table.
+     *
+     * @param  string  $column
+     * @return string
+     */
+    public function qualify($column)
+    {
+        if (Str::contains($column, '.')) {
```

Comment:
What is this? What if my column name actually has dots in it?

Reply:
Do you have an example for a column with dots?

Reply:
What example do you need? Any name with dot. user.name

Reply:
Maybe we can add a comment here, something like:
> Don't qualify the column if a `.` is found, in which case it's assume to be qualified already

I just hope (not an SQL expert here) it's not possible to have a actual column name with a dot ... in which case the worst output would be that the qualification wouldn't be performed and the user would have to do it manually 🤷‍♀️